README.md

```diff
@@ -1,109 +1,67 @@
+pa11ycrawler a [Scrapy](http://doc.scrapy.org/en/latest/index.html)
+spider that calls [Pa11y](http://pa11y.org/) on every page of a locally-running
+Open edX installation, to audit it for accessibility purposes.
+It will store the result of each page audit in a data directory as a set of
+JSON files, which can be transformed into a beautiful HTML report.
 
 Installation
 ============
 
-Prereqs
--------
-
-* python (v2.7)
-* node.js
-
-Install node Requirements
--------------------------
-```
-npm install -g pa11y@3.6.0 pa11y-reporter-1.0-json@1.0.2
-```
+pa11ycrawler requires Python 2.7 and Node.js installed.
 
-Install via GitHub
-------------------
 ```
-pip install git+https://github.com/edx/pa11ycrawler.git
+pip install .
+npm install
 ```
 
-Install via PyPi
-----------------
-TODO
-
-
 Usage
 =====
 
-Basic usage
------------
-For help:
-
-```
-pa11ycrawler -h
-```
-
-Running the crawler
--------------------
-
-To run the crawler, producing json reports
-```
-pa11ycrawler run $START_URL --pa11ycrawler-allowed-domains=$ALLOWED_DOMAINS --pa11y-reporter='1.0-json'
-```
-
-NOTE: You probably want to make sure that `--pa11ycrawler-allowed-domains` is set, or you may start crawling external sites.
-
-For more options:
 ```
-pa11ycrawler run -h
+scrapy crawl edx
 ```
 
+There are several options for this spider that you can configure using the
+`-a` scrapy flag.
 
-Produce html reports from the 1.0-json reports
-----------------------------------------------
+Option       | Default                        | Example
+------------ | ------------------------------ | -------
+`domain`     | `localhost`                    | `scrapy crawl edx -a domain=edx.org`
+`port`       | `8000`                         | `scrapy crawl edx -a port=8003`
+`course_key` | `course-v1:edX+Test101+course` | `scrapy crawl edx -a course_key=org/course/run`
+`data_dir`   | `data`                         | `scrapy crawl edx -a data_dir=~/pa11y-data`
 
+These options can be combined by specifying the `-a` flag multiple times.
+For example, `scrapy crawl edx -a domain=courses.edx.org -a port=80`.
 
-To run the crawler, producing json reports
-```
-pa11ycrawler json-to-html
-```
-
-For more options:
-```
-pa11ycrawler json-to-html -h
-```
-
-
-Development
-===========
-
-Prereqs
--------
+The `data_dir` option is used to determine where this crawler will save its
+output. pa11ycrawler will run each page of the site through `pa11y`,
+encode the result as JSON, and save it as a file in this directory.
+This data directory is "data" by default, which means it will create a directory
+named "data" in whatever directory you run the crawler from.
+Whatever directory you specify, it will be automatically created if it does
+not yet exist. In addition, this tool will never delete data from the data
```

Comment:
There's got to be some way of programmatically cleaning out your results. I had asked about this in the `clean` task, and you pointed out that we can't specify a data directory. Here you point out that we shouldn't clean it as a natural part of a run. But we must have a way of clearing out old results, and that should be any one of the following: a make task, a pre-run task, a specific clean-my-data-dir task. I can understand your arguments and will not push too hard on the specific examples; however, we must have a way of cleaning out the data through either the Makefile or the crawl tasks. At the least it will be much more convenient and at the most we can have it scripted as part of any sort of cron job.